virttest/utils_test/libvirt.py

```diff
@@ -3436,6 +3436,7 @@ def modify_vm_iface(vm_name, oper, iface_dict, index=0):
     del_addr = iface_dict.get('del_addr')
     del_rom = iface_dict.get('del_rom')
     del_filter = iface_dict.get('del_filter')
+    del_mac = iface_dict.get('del_mac')
```

Comment:
del_mac in your tests is set to "yes" or "no", how about change to:
del_mac = "yes" == iface_dict.get("del_mac", "no")

Reply:
> del_mac in your tests is set to "yes" or "no", how about change to:
> del_mac = "yes" == iface_dict.get("del_mac", "no")

no, the {"del_mac": "yes"} is an optional parameter just like the parameters above "del_rom" "del_addr", it no need to do the delete, there is no such parameter passed here.

Reply:
Yes, I understand.  But for a case which wants to keep mac address, if it's wrongly set to "no", it will delete the mac address with your code. 
>>> a="no"
>>> if a:
...  print("ds")
... 
ds
>>>

Reply:
updated